maas-schemas/schemas/core/personal-document.json

```diff
@@ -44,17 +44,19 @@
           "type": "string"
         }
       }
+    },
+    "media": {
```

Comment:
As discussed in DM, suggestion would be making it `{"content": "encoded-string", "type/context": "front"/"back"}`

Reply:
Thanks fixing.

Reply:
Fixed!